apis/dataform/v1alpha1/repository_types.go

```diff
@@ -68,9 +94,26 @@ type DataformRepositorySpec struct {
 	// +optional
 	ResourceID *string `json:"resourceID,omitempty"`
 
+	// Repository user labels. An object containing a list of "key": value pairs. Example: { "name": "wrench", "mass": "1.3kg", "count": "3" }.
```

Comment:
I don't think we should do these yet, today in KCC labels come from metadata.labels on the object.  I think what you're doing here is correct, it's just a bigger discussion!